polyopt/test_polyopt.m

```diff
@@ -1,48 +1,49 @@
-function test_suite = test_polyopt
-% function test_suite = test_polyopt
-%
-% A set of verification tests for the polyopt suite.
+% test_polyopt.m A set of verification tests for the polyopt suite.
 
-initTestSuite;
+% setup
+cvx_clear;
+tol = 1.e-2;
 
-function tol = setup
-    cvx_clear;
-    tol = 1.e-2;
+%% Test real axis monomial
+lambda = spectrum('realaxis', 100);
+s = randi(6);
+p = 1;
+[h, poly_coeff] = opt_poly_bisect(lambda, s, p, 'monomial');
+assert(all(ismembertol(h, 2*s^2, tol, 'ByRows', true)))
 
-function test_realaxis_monomial(tol)
-    lamda = spectrum('realaxis',100);
-    s=randi(6); p=1;
-    [h,poly_coeff]=opt_poly_bisect(lamda,s,p,'monomial')
-    assertElementsAlmostEqual(h,2*s^2,'relative',tol);
+%% Test real axis Chebyshev
+lambda = spectrum('realaxis', 100);
+s = randi(10);
+p = 1;
+[h, poly_coeff] = opt_poly_bisect(lambda, s, p, 'chebyshev');
+assert(all(ismembertol(h, 2*s^2, tol, 'ByRows', true)))
 
-function test_realaxis_chebyshev(tol)
-    lamda = spectrum('realaxis',100);
-    s=randi(10); p=1;
-    [h,poly_coeff]=opt_poly_bisect(lamda,s,p,'chebyshev')
-    assertElementsAlmostEqual(h,2*s^2,'relative',tol);
+%% Test imaginaly axis monomial
```

Comment:
imaginary

Reply:
I've fixed this typo in https://github.com/ranocha/RK-opt/commit/08f2c532d462b49cd1d080560ced74d8d52a371d, but the new commit does again not show up here... GitHub is really weird today.